### PR TITLE
updated to meet BPs - 301-vmss-windows

### PR DESCRIPTION
**Pull request by bmoore-msft**
### Description of the change
updates to meet BPs

### CODE REVIEW

File: 301-multi-vmss-windows/azuredeploy.json [hunk 1]
```diff
@@ -23,33 +28,32 @@
         "description": "Number of VM instances per scale set (max of 100)"
```
Comment (philon-msft) [inline 301-multi-vmss-windows/azuredeploy.json]: Should instanceCountPerVMSS and numberOfVMSS specify a minValue of 1 or 2?

Comment (bmoore-msft) [inline 301-multi-vmss-windows/azuredeploy.json]: will leave this as is for now

File: 301-multi-vmss-windows/azuredeploy.parameters.json [hunk 2]
```diff
@@ -2,17 +2,14 @@
...
-    },
     "numberOfVMSS": {
       "value": 5
     },
     "instanceCountPerVMSS": {
       "value": 5
     },
     "adminUsername": {
-      "value": "multi"
+      "value": "ghuser"
```
Comment (philon-msft) [inline 301-multi-vmss-windows/azuredeploy.parameters.json]: GEN-USER

**Comment by bmoore-msft** [comment]
@philon-msft - ok, I think I got everything on this one was well.  Can you have a look?

**Comment by bmoore-msft** [comment]
Ok, I think I got 'em... again...

**Pull request merged**